javascript/packages/orchestrator/src/chain-spec.ts

```diff
@@ -580,6 +580,57 @@ export async function getChainIdFromSpec(specPath: string): Promise<string> {
   });
 }
 
+export async function customizePlainRelayChain(
+  specPath: string,
+  networkSpec: ComputedNetwork,
+): Promise<void> {
+  // Relay-chain spec customization logic
+  const plainRelayChainSpec = readAndParseChainSpec(specPath);
+  const keyType = specHaveSessionsKeys(plainRelayChainSpec)
+    ? "session"
+    : "aura";
```

Comment:
Could be good to have an enum for this, maybe create a issue like the ZombieRole one ?